src/repositories/memory/HandleStore/index.ts

```diff
@@ -265,35 +248,37 @@ export class HandleStore {
             created_slot_number: slotNumber,
             updated_slot_number: slotNumber,
             hasDatum: !!datum,
-            datum: isDatumEndpointEnabled() && datum ? datum : undefined
+            datum: isDatumEndpointEnabled() && datum ? datum : undefined,
+            personalization
         };
 
         return newHandle;
     };
 
     static buildHandleHistory(
-        newHandle: IHandle,
-        oldHandle?: IHandle,
-        personalization?: IPersonalization
+        newHandle: Partial<IPersonalizedHandle>,
+        oldHandle?: Partial<IPersonalizedHandle>
     ): HandleHistory | null {
         const { name } = newHandle;
         if (!oldHandle) {
-            return NODE_ENV !== 'production' ? { old: null, new: { name } } : { old: null };
+            return NODE_ENV === 'production' ? { old: null } : { old: null, new: { name } };
```

Comment:
Why does this consider production?

Reply:
This is so we can check the `new` property during testing